### ISSUE TITLE
setup module fails on RHEL5.7 host with bonding

### ISSUE THREAD

**Issue by haskjold** [ansible/ansible#2813]
RHEL5 doesn't seem to have the file present that setup assumes for bonded interfaces.

``` Bash
$ ansible -k -K foo.bar.com -m setup --tree /tmp/facts/
foo.bar.com | FAILED => failed to parse: 
Traceback (most recent call last):
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 2561, in ?
    main()
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 1676, in main
    data = run_setup(module)
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 1621, in run_setup
    facts = ansible_facts()
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 1612, in ansible_facts
    facts.update(Network().populate())
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 1012, in populate
    interfaces, ips = self.get_interfaces_info(ip_path, default_ipv4, default_ipv6)
  File "/tmp/ansible-1367313467.37-186933245391615/setup", line 1099, in get_interfaces_info
    interfaces[device]['all_slaves_active'] = open(os.path.join(path, 'bonding', 'all_slaves_active')).read() == '1'
IOError: [Errno 2] No such file or directory: '/sys/class/net/bond0/bonding/all_slaves_active'
```

Target host

``` Bash
$ ls /sys/class/net/bond0/bonding/
active_slave    arp_ip_target  mii_status        slaves
ad_actor_key    arp_validate   mode              updelay
ad_aggregator   downdelay      num_grat_arp      use_carrier
ad_num_ports    fail_over_mac  num_unsol_na      xmit_hash_policy
ad_partner_key  lacp_rate      primary
ad_partner_mac  mark_steering  primary_reselect
arp_interval    miimon         resend_igmp

$ cat /etc/redhat-release 
Red Hat Enterprise Linux Client release 5.7 (Tikanga)
```

**Comment by mpdehaan** [comment]
Thanks, I'll fix this to test for file existence ASAP.

cc @dagwieers

**Comment by mpdehaan** [comment]
Fix pushed, I didn't have a bonded RHEL 5.7 setup so there may be some things further down the line -- let me know if you should hit any problems.

Thanks!
